js/dataframe/shared/table_utils.ts

```diff
@@ -0,0 +1,28 @@
+export type TableData = {
+	value: string | number;
+	id: string;
+}[][];
+
+export async function copy_table_data(data: TableData): Promise<void> {
+	const table_data = data
+		.map((row) => row.map((cell) => String(cell.value)).join("\t"))
```

Comment:
I think we should include the headers and use commas as separators so that it's like a csv